examples/ex17p.cpp

```diff
@@ -307,8 +308,10 @@ int main(int argc, char *argv[])
       x.Neg(); // x = -x
 
       ostringstream mesh_name, sol_name;
-      mesh_name << "mesh." << setfill('0') << setw(6) << mpi.WorldRank();
-      sol_name << "sol." << setfill('0') << setw(6) << mpi.WorldRank();
+      mesh_name << "mesh." << setfill('0')
+                << setw(6) << Mpi::Session().WorldRank();
+      sol_name << "sol." << setfill('0')
+               << setw(6) << Mpi::Session().WorldRank();
```

Comment:
Similar to my suggestion to shorten `Mpi::Session().Root()` --> `Mpi::Root()`, I suggest we also shorten `Mpi::Session().WorldRank()` --> `Mpi::WorldRank()`.